test/datasource/index.spec.ts

```diff
@@ -1,10 +1,13 @@
 import * as datasource from '../../lib/datasource';
 import * as _npm from '../../lib/datasource/npm';
+import * as _hostRules from '../../lib/util/host-rules';
 
 jest.mock('../../lib/datasource/docker');
 jest.mock('../../lib/datasource/npm');
+jest.mock('../../lib/util/host-rules');
 
 const npmDatasource: any = _npm;
```

Comment:
```suggestion
// TODO: fix types
const npmDatasource: any = _npm;
```

Please add as info for us

Reply:
This is required as the test is written in typescript ,and as such does not have some of the necessary methods and properties for it to be mockedvia jest, this was a workaround for that type error.